maven-publish.gradle

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
This file should nave a copyright notice, copied from `build.gradle`.

Reply:
Completely forgot about it. Added!